packages/blade/docs/guides/IntegrationExamples.stories.mdx

```diff
@@ -0,0 +1,10 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Guides/Integration examples" />
+
+# Integration examples
+
+You may checkout some examples on integrating `blade` in your project:
```

Comment:
@divyanshu013 i was mostly saying on the lines of adding a live playground or probably a codesandbox embed 😞 . For now we can merge this but for other upcoming integrations we shall create a directory on codesandbox too for such integrations